package.json

```diff
@@ -30,7 +30,7 @@
     "@dojo/has": "beta1",
     "@dojo/i18n": "beta1",
     "@dojo/shim": "beta1",
-    "maquette": "2.4.3"
+    "maquette": "2.5.0"
```

Comment:
We had had specific issues before with some compatibility.  Actually between `2.4.3` and `2.5.0`, there was a regression to some of our tests (#492).  It is risky to float it because it is such a fundamental dependency.